app/views/outreach_events/show.html.erb

```diff
@@ -39,5 +44,16 @@
   <%= @outreach_event.outreach_event_types.map(&:name).join(", ") %>
 </p>
 
-<%= link_to 'Edit', edit_outreach_event_path(@outreach_event) %>
-<%= link_to 'Back', outreach_events_path %>
\ No newline at end of file
+<% if current_user.admin? || @outreach_event.user_id == current_user.id %>
```

Comment:
Since this logic already lives in the OutreachEventPolicy, we can use that to our advantage! Instead of the above, we could do `if OutreachEventPolicy.new(current_user, @outreach_event).update?`